WordPress/src/main/java/org/wordpress/android/ui/reader/utils/ReaderXPostUtils.java

```diff
@@ -0,0 +1,73 @@
+package org.wordpress.android.ui.reader.utils;
+
+import android.net.Uri;
+import android.support.annotation.NonNull;
+import android.text.Html;
+import android.text.Spanned;
+
+import org.wordpress.android.models.ReaderPost;
+
+/**
+ * Reader cross-post utility routines
+ */
+
+public class ReaderXPostUtils {
+
+    // note that these strings don't need to be localized due to the intended audience
+    private static final String UNKNOWN_SITE = "(unknown)";
+    private static final String FMT_SITE_XPOST = "%1$s cross-posted from %2$s to %3$s";
+    private static final String FMT_COMMENT_XPOST = "%1$s left a comment on %2$s, cross-posted to %3$s";
```

Comment:
> I wonder if on mobile we could compact these phrases more? Maybe use x-posted and remove the + from the P2 name?

I like this - and perhaps also changing "left a comment on" to simply "commented on." @aerych Thoughts?